set_window_title.py

```diff
@@ -194,17 +137,10 @@ def rename_window_linux(self, window, official_title, new_title, debug=False):
         if output:
           print("[SetWindowTitle] Error: Failure when renaming:", output)
 
-  def rename_window_windows(self, window, official_title, new_title):
-    w = self.window_handle_cache.get(window.id(), None)
-    if w is None:
-      for w in list_all_windows():
-        if w.title.endswith(official_title):
-          w.title = new_title
-          # TODO: Understand why caching the windows handle cause crashes.
-          # self.window_handle_cache[window.id()] = w
-    else:
-      w.title = new_title
-
+  def rename_window_windows(self, new_title):
+    # PX_WINDOW_CLASS is the ClassName of SublimeText, can be seen via a tool such as Nirsoft WinLister
+    hwndSublime = ctypes.windll.user32.FindWindowA(b'PX_WINDOW_CLASS', None)
```

Comment:
I think this is probably what is causing #15.
If FindWindowA returns NULL, we will call SetWindowText(null, new_title) and probably crash.

Reply:
we can add a if(hwndSublime): below this line in this case instance